modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,136 @@ Module included in the following assemblies:
+
+=== Discovery selectors Support
+This release introduces a generally available version of the `meshConfig.discoverySelectors` field, which can be used in cluster-wide deployment to limit which services the Service Mesh Control Plane can discover.
+
+[source,yaml]
+----
+spec:
+  meshConfig    
+    discoverySelectors:
+    - matchLabels:
+        env: prod
+        region: us-east1
+    - matchExpressions:
+      - key: app
+        operator: In
+        values:
+          - cassandra
+          - spark
+----
+
+=== Integration with cert-manager istio-csr
+Beginning with this release, {SMProductName} integrates with `cert-manager` and `istio-csr`. `cert-manager` adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. `cert-manager` provides and rotates an intermediate CA certificate for Istio. Integration with `istio-csr` enables users to delegate signing certificate requests from Istio proxies to `cert-manager`.
+
+[NOTE]
+====
+Integration with `cert-manager` and 'istio-csr' is not supported on {ibmzProductName} or {linuxoneProductName}.
+====
+
+=== Integration with external authorization systems 
+This release introduces a generally available method of integrating {SMProductName} with external authorization systems using the `action: CUSTOM` field of the `AuthorizationPolicy` resource. Use `envoyExtAuthzHttp` to delegate the access control to an external authorization system.
+
+=== Integration with external Prometheus installation
+
+This release introduces a generally available version of the Prometheus extension provider. You can expose metrics to the OpenShift Monitoring stack or a custom Prometheus installation by setting the `extensionProviders` field to `prometheus` in the `spec.meshConfig` specification. The telemetry object configures Istio proxies to collect traffic metrics. {SMProductShortName} only supports the Telemetry API for Prometheus metrics. 
+
+[source,yaml]
+----
+spec:
+  meshConfig:
+    extenstionProviders:
+    - name: prometheus
+      prometheus: {}
+---
+apiVersion: telemetry.istio.io/v1alpha1
+kind: Telemetry
+metadata:
+  name: enable-prometheus-metrics
+spec:
+  metrics:
+  - providers:
+    - name: prometheus
+----
+
+=== IPv6 Single Stack Support
+:FeatureName: IPv6 Single Stack Support
+This release introduces a generally available version of IPv6 Single Stack Support, providing access to a broader range of IP addresses.
+
+[NOTE]
+====
+IPv6 is not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}.
+====
+
+=== {product-title} Gateway API Support
+:FeatureName: {product-title} Gateway API
+include::snippets/technology-preview.adoc[]
+This release introduces a Technology Preview version of {product-title} Gateway API Support. By default, the functionality is disabled.
```

Comment:
As this feature was present as a TP feature in previous versions of OSSM, I think it'd be more correct to say that "This release introduces and **updated** Technology preview version of...."